packages/orm/src/dataloaders/findOrCreateDataLoader.ts

```diff
@@ -66,7 +55,7 @@ export function findOrCreateDataLoader<T extends Entity>(
       // This is fundamentally asking to create the same entity, but with different ifNew/upsert
       // conditions, which we could fail on, but for now just take the first tuple of
       // {where/ifNew/upsert} and assume it wins
-      const [{ where, ifNew, upsert }] = keys;
+      const [{ ifNew, upsert }] = keys;
```

Comment:
I removed the `where` from the dataloader key, b/c we already have it as part of the batch key, and the batch key has the trimmed values.